recipes-debian/mhash/mhash_debian.bb

```diff
@@ -0,0 +1,34 @@
+inherit debian-package
+require recipes-debian/sources/mhash.inc
+
+LICENSE = "LGPLv2+"
+LIC_FILES_CHKSUM = "file://COPYING;md5=3bf50002aefd002f49e7bb854063f7e7"
+
+SECTION = "libs"
+
+SRC_URI += "file://Makefile.test \
+    file://mhash.c \
+    file://run-ptest \
+    "
+
+DEBIAN_UNPACK_DIR = "${WORKDIR}/mhash-${PV}.orig"
+
+inherit autotools pkgconfig ptest
+
+do_compile_prepend () {
+	sed -i 's:-I$(top_builddir)/include/mutils:-I$(top_srcdir)/include:g' lib/Makefile
+	sed -i 's:-I$(top_builddir)/include/mutils:-I$(top_srcdir)/include:g' src/Makefile
+}
+
+do_compile_ptest() {
+	if [ ! -d ${S}/demo ]; then mkdir ${S}/demo; fi
+	cp ${WORKDIR}/Makefile.test ${S}/demo/Makefile
+	cp ${WORKDIR}/mhash.c ${S}/demo/
+	cp include/mutils/mhash_config.h ${S}/include/mutils/
+	cp -r lib/.libs ${S}/lib/
+	make -C ${S}/demo CFLAGS="${CFLAGS} -I${S}/include/" LDFLAGS="${LDFLAGS} -L${S}/lib/.libs"
+}
+
+do_install_ptest() {
+	install -m 0755 ${S}/demo/mhash ${D}${PTEST_PATH}
+}
```

Comment:
I found meta-security has a corresponding recipe of libmhash.
https://layers.openembedded.org/layerindex/recipe/101469/

Could you refer to this recipe?

Reply:
OK. The header is added. Also, renamed to libmhash from mhash.